src/main/java/io/seqera/tower/cli/commands/workspaces/UpdateCmd.java

```diff
@@ -0,0 +1,36 @@
+package io.seqera.tower.cli.commands.workspaces;
+
+import io.seqera.tower.ApiException;
+import io.seqera.tower.cli.responses.Response;
+import io.seqera.tower.cli.responses.Workspaces.WorkspaceUpdated;
+import io.seqera.tower.model.DescribeWorkspaceResponse;
+import io.seqera.tower.model.OrgAndWorkspaceDbDto;
+import io.seqera.tower.model.UpdateWorkspaceRequest;
+import io.seqera.tower.model.Visibility;
+import picocli.CommandLine;
+
+import java.io.IOException;
+
+@CommandLine.Command(
+        name = "update",
+        description = "Update an existing organization's workspace"
+)
+public class UpdateCmd extends AbstractWorkspaceCmd {
```

Comment:
I find two problems here:
1. If we add an `id` optional parameter, the `name` parameter will have to be optional as well. Doing so will make all parameters optional.
2. There is no endpoint for workspace deletion by `id` only. It requires the organization `id` as well. This makes the use of a single `id` parameter senseless rather than convenient.